lib/cli/writer.js

```diff
@@ -108,7 +115,12 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
           return Bluebird.resolve();
         }
 
-        return mountutils.unmountDiskAsync(drive.device);
+        // Because Mac OS doesn't like immediate unmounting / ejection
+        // after closing the file handle, as it's busy remounting,
```

Comment:
Did we ever track down _why_ Mac OS is doing this remounting?  (and is it for all images, or only particular images?)
I think I asked somewhere else if you were able to verify that writing an image with `dd` also results in the same auto-remounting behaviour?